### PR TITLE
Add missing bullets on the footer

### LINKED ISSUE #54: Missing bullets in assignment page footer
**Describe the bug**
In the assignment page footer, the bullets are missing.

**Expected behavior**
Screenshot 2020-04-06 at 22 38 20

**Now**
Screenshot 2020-04-06 at 22 39 20

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #54 

## Approach
- Edit the relevant .html file.
- Add the missing code lines

### Screenshots
Screenshot 2020-04-06 at 22 46 22

### Preview Link
https://kumuditha-udayanga.github.io/sack-site/apps/assignments/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/assignments/index.html [hunk 1]
```diff
@@ -277,29 +277,23 @@ <h3>Subscribe</h3>
                 <h4>Discover</h4>
                 <div class="separator"></div>
                 <ul class="list-block">
-                    <li>
-                        <a href="../../anthem.html">College Crest</a>
-
+                    <li><i class="fa fa-angle-right"></i>
+                        <a href="anthem.html">College Crest</a>
```
[the remark this answers is not in the document]
Comment (kumuditha-udayanga) [reply apps/assignments/index.html:281]: sure!

**Comment by Piumal1999** [comment]
@all-contributors please add @kumuditha-udayanga for code

File: apps/assignments/index.html [hunk 2]
```diff
@@ -277,29 +277,23 @@ <h3>Subscribe</h3>
                 <h4>Discover</h4>
                 <div class="separator"></div>
                 <ul class="list-block">
-                    <li>
+                    <li><i class="fa fa-angle-right"></i>
                         <a href="../../anthem.html">College Crest</a>
-
                     </li>
```
Comment (jayasanka-sack) [inline apps/assignments/index.html:282]: Could you please format this as following?

```html
<li>
    <i class="fa fa-angle-right"></i>
    <a href="../../anthem.html">College Crest</a>
</li>
```

Comment (kumuditha-udayanga) [reply apps/assignments/index.html:282]: ok

Comment (kumuditha-udayanga) [reply apps/assignments/index.html]: done

**Pull request merged**